recipe/meta.yaml

```diff
@@ -42,6 +42,7 @@ about:
   license_file: LICENSE
   summary: 'Plug-in for flake 8 to check the PEP-8 naming conventions'
   dev_url: https://github.com/PyCQA/pep8-naming
+  doc_url: https://github.com/PyCQA/pep8-naming/blob/main/README.rst
```

Comment:
hmm, for a rebuild, you would need to bump build-number, or was this package not biult before in that version?